### ISSUE TITLE
Reason for including assets is unclear

### ISSUE THREAD

**Issue by ivalentine** [learn-co-curriculum/crud-with-validations-lab#25]
Step 6 of the instructions states:

> Be sure to [include your new assets](http://apidock.com/rails/v4.2.1/ActionView/Helpers/AssetTagHelper/javascript_include_tag) in `application.html.erb`

As far as I can tell, none of the tests require JavaScript, so the reason for doing this is unclear. Is it just to get students accustomed to using Rails helpers, or is it actually required for some reason?

Reported by @sdcrouse.

**Comment by Sdcrouse** [comment]
Adding to that, when I added `<%= javascript_include_tag 'songs.js' %>` to my `application.html.erb` file, I got this error:

"Sprockets::Rails::Helper::AssetNotPrecompiled in Songs#index
Asset was not declared to be precompiled in production. 
Add `Rails.application.config.assets.precompile += %w( songs.js )` to `config/initializers/assets.rb` and restart your server."

I added that line of code to `assets.rb`, and my program worked. I hope that helps!

---Sdcrouse
